docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -971,8 +971,8 @@ concat_ws(separator, str[, ..., str_n])
 
 #### Arguments
 
-- **separator**: Separator to insert between concatenated strings.
 - **str**: String expression to operate on. Can be a constant, column, or function, and any combination of operators.
```

Comment:
I think `separator` should come first in order like it is in the function signature

Reply:
I tried changing the order between argument and standard argument in user_doc(), but the documentation order didn't change. Is the order of content sections fixed?

Reply:
I will check that